src/app/components/community/sidebar/sidebar-data.ts

```diff
@@ -0,0 +1,32 @@
+import { Community } from '../../../../_common/community/community.model';
+import { User } from '../../../../_common/user/user.model';
+
+export class CommunitySidebarData {
```

Comment:
Why the need for all the `!` in the properties? When is payload ever undefined? It feels really messy to have the type annotations being potentially wrong...